### PR TITLE
docs: add routing_algo example

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: docs/user/site.rst [hunk 1]
```diff
@@ -235,15 +235,19 @@ mesh
     in the *batman_adv* section:
 
     The optional value *gw_sel_class* sets the gateway selection class. The
-    default is class 20, which is based on the link quality (TQ) only; class 1
+    default is class ``20``, which is based on the link quality (TQ) only; class ``1``
     is calculated from both the TQ and the announced bandwidth.
+
+    The optional value *routing_algo* allows to set up ``BATMAN V``-based meshes.
```
Comment (mweinelt) [inline docs/user/site.rst]: Use `BATMAN_V` (with underline) if you put them in brackets.

File: docs/user/site.rst [hunk 2]
```diff
@@ -235,15 +235,19 @@ mesh
     in the *batman_adv* section:
 
     The optional value *gw_sel_class* sets the gateway selection class. The
-    default is class 20, which is based on the link quality (TQ) only; class 1
+    default is class ``20``, which is based on the link quality (TQ) only; class ``1``
     is calculated from both the TQ and the announced bandwidth.
+
+    The optional value *routing_algo* allows to set up ``BATMAN V``-based meshes. 
+    If unset, the routing algorithm will default to ``BATMAN IV``.
```
Comment (mweinelt) [inline docs/user/site.rst]: same as previous

File: docs/user/site.rst [hunk 3]
```diff
@@ -235,15 +235,19 @@ mesh
...
     ::
 
       mesh = {
         vxlan = true,
         filter_membership_reports = false,
         batman_adv = {
           gw_sel_class = 1,
+          routing_algo = 'BATMAN_V',
```
Comment (mweinelt) [inline docs/user/site.rst]: Please set this to `BATMAN_IV`, as this is the current default. If people blindly copy from the documentation and have no clue what they're doing that is probably not what they're expecting. Instead follow the principle of least surprise.

**Comment by mweinelt** [comment]
Freifunk Chemnitz has a site deployed using BATMAN_V afaik.

**Comment by rubo77** [comment]
I see no batman Vs in the Chemnitz config https://gitlab.com/FreifunkChemnitz/site-ffc/

**Comment by mweinelt** [comment]
Merged in 2b066a34a49b72348cd5edeb85f2c4c6c73bcf01. Thanks!

**Pull request closed without merging**